src/sgame/sg_spawn_gfx.cpp

```diff
@@ -137,6 +137,12 @@ static void findEmptySpot( glm::vec3 const& origin, float radius, glm::vec3& spo
 
 void SP_gfx_light_flare( gentity_t *self )
 {
+	if ( !self->shaderKey )
+	{
+		Log::Warn( "Light flare entity at (%d, %d, %d) has no shader", self->s.origin[0], self->s.origin[1], self->s.origin[2] );
```

Comment:
Consider including entity number as a better way of identifiying entities (in addition to the location).

Reply:
Done.